modules/nw-ingress-route-secret-load-external-cert.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * ingress/routes.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nnw-ingress-route-secret-load-external-cert_{context}"]
+= Creating Route with Externally Managed Certificate
+
+You can now seamlessly integrate OpenShift Routes with third-party certificate management solutions. The extended Route API enables you to reference externally managed TLS certificates via secrets, eliminating the need for manual certificate management and reducing potential errors. This feature ensures a smoother rollout of certificate updates, enabling the OpenShift router to serve renewed certificates promptly.
```

Comment:
I'm not aware of the docs standard for words like: `Routes`, `Route`, `API` : do they start with capital letters?